components/Footer.tsx

```diff
@@ -32,17 +49,43 @@ const OSLLogo = () => (
       width="24px"
       height="32px"
       layout="fixed"
+      className="hover:brightness-50"
```

Comment:
This is just a bit of a hack until we have an SVG logo we can use here that will inherit a font colour.